sdk/Javascript/src/Platforms/Platform.FxOS.js

```diff
@@ -0,0 +1,277 @@
+// ----------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ----------------------------------------------------------------------------
+
+/// <reference path="..\Generated\MobileServices.DevIntellisense.js" />
+/*global $__fileVersion__:false, $__version__:false */
+
+var _ = require('Extensions');
+var Validate = require('Validate');
+var Promises = require('Promises');
```

Comment:
I implemented Platform.FirefoxOS.js on the basis of Platform.Web.js. And, as it was pointed out, it contains a few unnecessary code.
Should I change to simple code by using build in Promise object and send pull request it again?